Makefile

```diff
@@ -35,6 +35,6 @@ GO=go
 GOFLAGS=""
 
 test:
-	@(./test/shell-pipeline-tests/openvino-bike-rentals/pipelines-test-openvino-bike-rentals.sh)
-	@(./test/shell-pipeline-tests/tensorflow-housing/pipelines-test-tensorflow-housing.sh)
+	@(./test/shell-pipeline-tests/seldon-bike-rentals/pipelines-test-seldon-bike-rentals.sh)
+	@(./test/shell-pipeline-tests/openvino-tensorflow-housing/pipelines-test-openvino-tensorflow-housing.sh)
 	@(cd test/e2e-tests/tests && ${GO} test)
```

Comment:
There is a conflict after the previous PR was merged.

Reply:
Fixed, thanks :+1: